src/shell.rs

```diff
@@ -19,19 +19,10 @@ enum ShellExecResult {
 
 fn shell_exec(vm: &VirtualMachine, source: &str, scope: Scope) -> ShellExecResult {
     match vm.compile(source, compile::Mode::Single, "<stdin>".to_owned()) {
-        Ok(code) => {
-            match vm.run_code_obj(code, scope.clone()) {
-                Ok(value) => {
-                    // Save non-None values as "_"
-                    if !vm.is_none(&value) {
-                        let key = "_";
```

Comment:
This is no longer required? This would break the interactive mode in which we can use the last result as the underscore right?

Reply:
Ignore my comment, I continued reading further down the change.